src/libudev/libudev.c

```diff
@@ -176,6 +177,7 @@ _public_ struct udev *udev_new(void) {
                                         log_set_max_level(prio);
                                 continue;
                         }
+#endif
```

Comment:
given that we only read a single config option from this I think we should skip the whole opening+parsing of the config file if the macro isn#t defined.